modules/ossm-rn-new-features.adoc

```diff
@@ -19,11 +19,101 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 2.2
+
+This release of {SMProductName} adds new features and enhancements, and is supported on OpenShift Container Platform 4.9, and 4.10.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
+
+|===
+|Component |Version
+
+|Istio
+|1.12.7
+
+|Envoy Proxy
+|1.20.4
+
+|Jaeger
+|1.30.2
+
+|Kiali
+|1.48.0.16
+|===
+
+=== `WasmPlugin` API
+This release changes the implementation of `WebAssembly` extensions to use the `WasmPlugin` API instead of the `ServiceMeshExtention` API, which has been deprecated.
```

Comment:
Unless I'm misunderstanding, this is incorrect.  This release adds support for the `WasmPlugin` API  and deprecates the `ServiceMeshExtention` API. The way this is phrased (changes the implementation) makes it sound like something is happening automatically, when there are actually manual steps that customers need to take.

Reply:
Agreed - perhaps it could be worded better to make that clearer, as indeed, it's the addition of a new API rather than a change of implementation. "This release adds support for the `WasmPlugin` API and deprecates the `ServiceMeshExtension` API. As this will be removed in an upcoming release of OpenShift Service Mesh, users should migrate to the new API as soon as possible". Or you can leave out the second part if you prefer a brevity.

Reply:
@longmuir, I'm happy to change the first part, and prefer brevity. I've always been encouraged not to mention things might happen in the future. 

I've made the change locally, but haven't submitted it yet.